### PR TITLE
New version development for auro-card

### LINKED ISSUE #44: auro-card: 3.0 - Base component.
## Is your feature request related to a problem? Please describe.
Create the basic version of `auro-card` 3.0 based off of the updated blueprint.

Blueprint: https://www.figma.com/file/Qi9dapT6aEd4EQUnWZW6Uz/Card?node-id=1721%3A5834&mode=dev

## Describe the solution you'd like
There will be two variations of this version, one with a border and one without.

## Describe alternatives you've considered
Create custom html cards any time we need them.

## Additional context
This is the first story of a full rewrite of this component.  All current API features will be deprecated and follow up stories will be created for future features.

### LINKED ISSUE #45: auro-card: navigation card
## Is your feature request related to a problem? Please describe.
A version of `auro-card` that is used for navigation where the entire card is clickable.

Blueprint: https://www.figma.com/file/Qi9dapT6aEd4EQUnWZW6Uz/Card?node-id=1721%3A5834&mode=dev

## Describe the solution you'd like
A new variation that can be toggled on via an attribute on the component.

## Describe alternatives you've considered
Use the base component instead of the nav style.

### PR DESCRIPTION
**Pull request by rsstdd**
# Alaska Airlines Pull Request

This is a rewrite of the Auro-Card component following the design guidelines outlined in [this Figma project](https://www.figma.com/file/Qi9dapT6aEd4EQUnWZW6Uz/Card?type=design&node-id=322-298&mode=design&t=Rd9H98ZTiiOMExrm-0). 

**Resolves:** # (issue, if applicable)

- Resolves #44 
- Resolves #45 

## Type of change:
- [x] Revision of an existing capability
- [x] Other (please elaborate)
  - Redesign of Card component. Breaking changes to API. 

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by rsstdd** [comment]
## Open Questions
1. If an end user uses an auro-header, the margin defined in that component creates additional space outside of the design spec. Do we override or let the consumer set that spacing?
2. Button or link?
3. Should this component be responsible for maintaining aspect ratios for images?
4. Should we Expose `css part` for .content?
5. Should this component be responsible for preventing layout shifts for images?
6. Do we want to set default dimensions for round images?

@blackfalcon

**Comment by braven112** [comment]
@blackfalcon What are we waiting on before this is merged and we have our new version released?

**Comment by blackfalcon** [comment]
> ## Open Questions
> 1. If an end user uses an auro-header, the margin defined in that component creates additional space outside of the design spec. Do we override or let the consumer set that spacing?
> 2. Button or link?
> 3. Should this component be responsible for maintaining aspect ratios for images?
> 4. Should we Expose `css part` for .content?
> 5. Should this component be responsible for preventing layout shifts for images?
> 6. Do we want to set default dimensions for round images?
> 
> @blackfalcon

1. Regardless if the user uses auro-header or a heading tag, the spacing is defined in the header CSS and will always apply. I am opting to use natural spacing versus the spec. I will address this with design when available. 
2. This will be up to the user to decide. 
3. No
4. Yes
5. No, incalculable options 
6. No, replaced custom CSS with auro-avatar

**Review by blackfalcon — DISMISSED** [review]

File: README.md
Comment (jordanjones243) [inline README.md]: Use cases need to be updated

File: demo/apiExamples.md
Comment (jordanjones243) [inline demo/apiExamples.md:1]: File needs to be named `api.md`

File: README.md
Comment (jordanjones243) [inline README.md:121]: Version needs to be updated

File: demo/api.html
Comment (jordanjones243) [inline demo/api.html:6]: Revert change

File: docs/partials/index.md
Comment (jordanjones243) [inline docs/partials/index.md:2]: Revert change

File: package.json
Comment (jordanjones243) [inline package.json:10]: Update version

File: scripts/generateDocs.mjs
Comment (jordanjones243) [inline scripts/generateDocs.mjs:8]: Revert change as this component is `esm` compatible

File: README.md
Comment (blackfalcon) [reply README.md]: See https://github.com/AlaskaAirlines/auro-card/pull/56/files#diff-fbf9bf670db9967a32915ed264a8b2637370e3d768ee7cb126124266ce965626

File: README.md
Comment (blackfalcon) [reply README.md:121]: This is dynamic content that will be updated with the commit back from the release.

File: package.json
Comment (blackfalcon) [reply package.json:10]: This will be updated with the semantic-release commit back to the repo after the release. We do not manually edit this value.

**Review by jordanjones243 — DISMISSED** [review]

**Comment by blackfalcon** [comment]
Link for @leeejune design review 

- https://alaskaairlines-auro-card-pr-56.surge.sh/

**Comment by blackfalcon** [comment]
> Link for @leeejune design review
> 
> * https://alaskaairlines-auro-card-pr-56.surge.sh/

@leeejune the link above has been updated per meeting action items.

**Review by leeejune — APPROVED** [review]

**Review by jordanjones243 — CHANGES_REQUESTED** [review]
Nitpick: I still see a couple comments referencing `demo.md` instead of `index.md` and `apiExamples.md` instead of `api.md`

**Review by jordanjones243 — APPROVED** [review]

**Pull request merged**
